src/Umbraco.Web.UI.Client/src/views/propertyeditors/datepicker/datepicker.controller.js

```diff
@@ -72,14 +130,16 @@ function dateTimePickerController($scope, notificationsService, assetsService, a
 
         var filesToLoad = ["lib/moment/moment-with-locales.js",
 						   "lib/datetimepicker/bootstrap-datetimepicker.js"];
-
-            
+        
 		$scope.model.config.language = user.locale;
-		
 
 		assetsService.load(filesToLoad, $scope).then(
             function () {
-				//The Datepicker js and css files are available and all components are ready to use.
+                //The Datepicker js and css files are available and all components are ready to use.
+
+                // Set min and max date and parse date from dateformat and locale http://momentjs.com/docs/#/parsing/
+                $scope.model.config.minDate = moment($scope.model.config.minDate, ["DD/MM/YYYY HH:mm:ss"], $scope.model.config.language);
+                $scope.model.config.maxDate = moment($scope.model.config.maxDate, ["DD/MM/YYYY HH:mm:ss"], $scope.model.config.language);

```

Comment:
Right now it doesn't work correctly because values in minimum and maximum date properties are changes from "2015-12-28T00:00:00" and "2015-12-30T00:00:00"  to "28/12/2015 00:00:00" and "30/12/2015 00:00:00" ... I am not sure how the best way is the handle the date input format. Futhermore all dates seems to be disabled now when minimum and maximum date are empty, I think it is because of the invalid dates.

If I before L140 enter:
```
$scope.model.config.minDate = "2015-12-28T00:00:00";
$scope.model.config.maxDate = "2015-12-30T00:00:00";
```
you should only be able to select "December 28, 29 and 30".